.github/workflows/close-conflicted-prs.yml

```
@@ -28,15 +28,34 @@ jobs:
+
+                let conflictStartTime = new Date(pr.updated_at);
+                
+                for (const event of timeline.data) {
+                  if (event.event === 'cross-referenced' && event.commit_id) {
+                    conflictStartTime = new Date(event.created_at);
+                    break;
+                  }
+                }
+
+                const conflictAgeDays = (now - conflictStartTime) / (1000 * 60 * 60 * 24);
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)